### PR TITLE
ENH: V71 release changes

### PR DESCRIPTION
**Pull request by mbechtold**
This PR includes all Fortran code changes related to the V71 release. 
Test results (DEBUG=0 and DEBUG=1):
CassavaTest.PRO
--> zero-diff with pascal executable
Alf2Isp50.PRM
--> rounding differences (last digit) with pascal executable for WC12 (and related variables) in all time steps which I think was observed in an earlier corner case test (@lbusschaert?)
Ottawa.PRM: rounding differences (last digit) for some time steps 

ToDo: 
Updating of V71 release notes: Include bug fix related to the reading of program parameters

### CODE REVIEW

File: src/startunit.F90 [hunk 1]
```diff
@@ -638,9 +649,69 @@ subroutine InitializeProject(iproject, TheProjectFile, TheProjectType)
...
+            if (.not. FileOK%OffSeason_Filename) then
+                write(tempstring, '(4a)') '               Off-season (OFF), '
+                call fProjects_write(trim(tempstring))
+            end if
+            if (.not. FileOK%Observations_Filename) then
+                write(tempstring, '(4a)') '               Field data (OBS), '
+                call fProjects_write(trim(tempstring))
+            end if
+            write(tempstring, '(4a)') '          - Check file Name(s), Path(s) &
+                               or Structure of project file.'
```
Comment (lbusschaert) [inline src/startunit.F90:714]: Was this check also tested in Fortran? E.g. run with a missing/erroneous path to see if it returns the right message.

**Review by lbusschaert — CHANGES_REQUESTED** [review]
It all looks good! I left a few minor comments.
I was just wondering whether you tested the new option that specifies the missing files, i.e. tried a run with a missing/erroneous path in the .PRM or .PRO file to check whether it returns the same message as in Pascal. 
I could also quickly test it.

File: src/global.f90 [hunk 2]
```diff
@@ -3421,8 +3435,8 @@ subroutine LoadManagement(FullName)
         end if
         read(fhandle, *) TempInt  ! Canopy cover (%) after cutting
         call SetManagement_Cuttings_CCcut(TempInt)
+        ! Next line NO LONGER CONSIDERED since AquaCrop version 7.1
```
Comment (gdelannoy) [inline src/global.f90:3438]: This comment might be confusing, since the next line is removed instead of commented?

File: src/interface_global.pas [hunk 3]
```diff
@@ -192,7 +193,6 @@ interface
      rep_Cuttings = Record
          Considered : BOOLEAN;
```
Comment (gdelannoy) [inline src/interface_global.pas:194]: Just happened to notice the uppercase letters, whereas all others are lowercase

**Review by gdelannoy — APPROVED** [review]
Minor cosmetic questions. I agree w/ Louise's comments.

File: src/global.f90 (hunk 2 above)
Comment (lbusschaert) [reply src/global.f90:3438]: I think that means that the line is still in the input file ( .MAN file I guess). So we should still read the line to get to the next one, but this parameter is not considered anymore, hence L3425 was removed.

File: src/interface_global.pas [hunk 4]
```diff
@@ -4909,6 +4921,7 @@ function SeasonalSumOfKcPot(constref TheDaysToCCini,TheGDDaysToCCini : integer;
 function HarvestIndexDay(constref DAP  : LongInt;
```
[the remark this answers is not in the document]
Comment (mbechtold) [reply src/interface_global.pas:4921]: I don't think it's duplicate. Here it has begin end and in L 4326 it's just the initial function declaration, right? To be honest, I don't recall all the pascal interface logic anymore ... in the hope we will never need it again.

File: src/run.f90 [hunk 5]
```diff
@@ -6401,17 +6377,19 @@ end subroutine AdjustSWCRootZone
...
     real(dp), intent(inout) :: Bin
     real(dp), intent(inout) :: Bout
     real(dp), intent(inout) :: AssimToMobilize
     real(dp), intent(inout) :: AssimMobilized
     real(dp), intent(inout) :: FracAssim
     logical, intent(inout) :: StorageON
     logical, intent(inout) :: MobilizationON
+    logical, intent(in) :: HarvestNow
 
     integer(int32) :: c1, c2
```
[the remark this answers is not in the document]
Comment (mbechtold) [reply src/run.f90:6391]: Correct, removed in next push. Thx.

File: src/run.f90 [hunk 6]
```diff
@@ -6401,17 +6377,19 @@ end subroutine AdjustSWCRootZone
...
-    real(dp) :: x
+    real(dp) :: x, FracSto, tmob
```
[the remark this answers is not in the document]
Comment (mbechtold) [reply src/run.f90:6392]: Thx, removed now.

File: src/startunit.F90 (hunk 1 above)
Comment (mbechtold) [reply src/startunit.F90:714]: Yes, I tested my implementation. I added now additionally a print to screen information that this information on missing files is written into OUTP/ListProjectsLoaded.OUT

File: src/interface_global.pas (hunk 4 above)
[the remark this answers is not in the document]
Comment (lbusschaert) [reply src/interface_global.pas:4921]: Right! Sorry, it's been a while

File: src/startunit.F90 (hunk 1 above)
Comment (lbusschaert) [reply src/startunit.F90:714]: Great!

File: src/interface_global.pas (hunk 3 above)
Comment (mbechtold) [reply src/interface_global.pas:194]: That's inconsistent in many places. Would keep it here and elsewhere for now since we dropped the interface anyway for now.

**Review by lbusschaert — APPROVED** [review]

**Comment by mbechtold** [comment]
I accounted / replied to all your comments. Since you both approved, I would merge the PR next.
